scalding-core/src/main/scala/com/twitter/scalding/examples/WordCountJob.scala

```diff
@@ -3,8 +3,9 @@ package com.twitter.scalding.examples
 import com.twitter.scalding._
 
 class WordCountJob(args: Args) extends Job(args) {
-  TextLine(args("input")).read.
-    flatMap('line -> 'word) { line: String => line.split("\\s+") }.
-    groupBy('word) { _.size }.
-    write(Tsv(args("output")))
+  TypedPipe.from(TextLine(args("input")))
+    .flatMap { line: String => line.split("\\s+") }
+    .map (_ -> 1L)
+    .sumByKey
+    .write(TypedTsv[(String, Long)](args("output")))
```

Comment:
I would have relied on the implicit back to typed pipe and pretended it was back as the pipe. But yes, might be better without...